membersuite_api_client/tests/test_memberships.py

```diff
@@ -21,6 +24,14 @@ def test_get_membership_for_org(self):
                          '6faf90e4-0074-cbb5-c1d2-0b3c539859ef')
 
         # Test org without a membership
+
+        # Need an Organization ID for a non-member org. The one here
+        # is the same used above for testing with a member org.
+        #
+        #            HERE!
+        #              |
+        #              |
+        #              V
         test_org_id = "6faf90e4-0007-c9dc-98b7-0b3c53985743"
```

Comment:
The comment below (Test org with a membership) makes it look like '6faf90e4-0007-c578-8310-0b3c53985743' is for an org with memberships, too.  Not so?


```python
        # Test org with a membership
        test_org_id = "6faf90e4-0007-c578-8310-0b3c53985743"
        membership_list = self.service.get_memberships_for_org(test_org_id)
        self.assertEqual(type(membership_list[0]), Membership)
        self.assertEqual(membership_list[0].id,
                         '6faf90e4-0074-cbb5-c1d2-0b3c539859ef')

        # Test org without a membership
        test_org_id = "6faf90e4-0007-c9dc-98b7-0b3c53985743"
```